### PR TITLE
Add order to the db

### PR DESCRIPTION
**Pull request by Svitlana-G**

### CODE REVIEW

File: frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx [hunk 1]
```diff
@@ -26,9 +32,66 @@ const OrderInfo = ({ orderData }) => {
     return quantity;
   }, [cartItems]);
 
-  // const sendOrder = () => {
-  //   console.log(cartItems, orderData);
-  // };
+  const sendOrder = async () => {
+    const orderItems = cartItems.map((item) => {
+      if (item.orderItemsCompositions) {
+        const bouquetItems = item.orderItemsCompositions.map(
```
Comment (halynarydasova) [inline frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx:38]: Would it be smarter to use this structure when passing data to the cart?

File: frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx [hunk 2]
```diff
@@ -26,9 +32,66 @@ const OrderInfo = ({ orderData }) => {
...
+        orderData.contactData.recipientLastName +
+        ' ' +
+        orderData.contactData.recipientFirstName +
+        ' ' +
+        orderData.contactData.recipientMiddleName;
+      receiverPhone = orderData.contactData.recipientPhone;
+    }
+
+    const result = await addOrderToDB({
+      postcardText: orderData.postcardData.postcardText,
```
Comment (halynarydasova) [inline frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx:75]: Maybe it's better to use destructuring here. The code seems too complex

File: frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx (hunk 1 above)
Comment (Svitlana-G) [reply frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx:38]: Could you please explain what you mean?

File: frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx (hunk 2 above)
Comment (Svitlana-G) [reply frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx]: Done

**Review by halynarydasova — APPROVED** [review]

File: frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx [hunk 3]
```diff
@@ -26,9 +30,80 @@ const OrderInfo = ({ orderData }) => {
...
+  const getReceiverInfo = () => {
+    let receiverName = '';
+    let receiverPhone = '';
+
+    if (orderData.contactData.recipient === 'I') {
+      receiverName = orderData.contactData.clientFirstName;
+      receiverPhone = formattedPhone(orderData.contactData.clientPhone);
+    } else {
+      receiverName =
+        orderData.contactData.recipientLastName +
```
[the remark this answers is not in the document]
Comment (Svitlana-G) [reply frontend/src/components/Order/components/OrderInfo/OrderInfo.jsx]: done

**Pull request merged**